src/openrct2/core/FlagHolder.hpp

```diff
@@ -9,6 +9,8 @@
 
 #pragma once
 
+#include "../util/Util.h"
```

Comment:
FlagHolder uses EnumToFlag and EnumToFlags, which are defined in Util.h. I forgot to include it on the initial PR, because the file where I ended up using it happened to include it as well.

Reply:
I'm not really a fan of including `Util.h` in headers. I suggest we move `EnumValue`, `EnumToFlag`, `EnumToFlags` to another header in another PR.